R/get_undata.R

```diff
@@ -0,0 +1,177 @@
+
+#' @title Scrape data.un.org tables
+#'
+#' @description Scrape multiple pages from any data.un.org table.
+#'
+#' @inheritParams polite::scrape
+#' @param pages \[`numeric()`\]\cr
+#'   Pages to scrape from the table. See `get_undata_npages` to scrape total
+#'   number of tables. Default is 1.
+#'
+#' @return \[`tibble()`\] all pages from the table combined together.
+#'
+#' @details
+#' A list of all possible tables is shown here http://data.un.org/Explorer.aspx.
```

Comment:
Can you include here or in the examples section a sampling of tables the demographics team might want to scrape?